circle.yml

```diff
@@ -14,9 +14,6 @@ jobs:
       - checkout
       - setup_remote_docker:
           version: 17.11.0-ce
-      - run:
```

Comment:
The idea behind this is that we could leverage docker's layer cache, but I'm not sure it was working.

Reply:
I see, we were doing it in prior to https://github.com/decidim/docker/pull/29/commits/483cf302b0548b955a1e3f22f9f7aa9be4ccc013. I removed it because it was not having any effect on build time, but forgot to remove the `docker pull`s. I think maybe basing images on other images that have been built in the same run was making it never hit?